deeplearning4j-nn/src/main/java/org/deeplearning4j/nn/layers/recurrent/LSTMHelpers.java

```diff
@@ -249,6 +251,15 @@ static public FwdPassReturn activateHelper(final Layer layer, final NeuralNetCon
                 currHiddenUnitActivations.muliColumnVector(timeStepMaskColumn);
                 currentMemoryCellState.muliColumnVector(timeStepMaskColumn);
             }
+           // boolean layerNormalization = true;
+            if (useLayerNormalization) {
+                Number stdev = currHiddenUnitActivations.stdNumber();
+                currHiddenUnitActivations = currHiddenUnitActivations.sub(Nd4j.mean(currHiddenUnitActivations));
+                if (stdev.floatValue() == stdev.floatValue()) {
+                    // don't divide by NaN. i.e., all zero activations.
+                    currHiddenUnitActivations = currHiddenUnitActivations.div(stdev);
+                }
```

Comment:
Maybe a small eps would be better here? Calculating the std deviation twice is expensive.